### PR TITLE
chore:fixing sign-in and removed jwt logs

### PR DESCRIPTION
**Pull request by Aditya-eddy**
when the user's logs in. After referesh it's logs out. This or corrects this.

### CODE REVIEW

**Review by khareyash05 — APPROVED** [review]
LGTM!

File: package-lock.json
Comment (khareyash05) [inline package-lock.json:1]: Revert this change, no need of version change, maybe try updating your branch with main

File: src/SignIn.ts
Comment (khareyash05) [inline src/SignIn.ts:1]: Address the semicolon issues, will cause havoc later while reviewing bigger PR's

**Review by khareyash05 — COMMENTED** [review]
Fix these and we are GTG!

**Review by khareyash05 — CHANGES_REQUESTED** [review]
Fix these and we are GTG!

File: src/SignIn.ts
Comment (Aditya-eddy) [reply src/SignIn.ts:1]: removed the semicolons.

File: package-lock.json
Comment (Aditya-eddy) [reply package-lock.json:1]: done.

File: src/extension.ts [hunk 1]
```diff
@@ -166,28 +166,30 @@ export function activate(context: vscode.ExtensionContext) {
...
 
+    let accessToken = context.globalState.get<string>('JwtToken');
 
     // Check if the access token is already present in the global state
-    const accessToken = context.globalState.get<string>('accessToken');
-    console.log("accessToken " , accessToken)
+    // const accessToken = context.globalState.get<string>('accessToken');
     // disable if access token is already present
     if (accessToken) {
+        console.log("accessToken is verfied user us logged in");
```
Comment (khareyash05) [inline src/extension.ts:184]: unecessary log

**Comment by khareyash05** [comment]
Not able to work on Arc browser limitations(will have to fix it soon), @Aditya-eddy if you could attach a video that would be great

**Review by khareyash05 — APPROVED** [review]
LGTM!

**Pull request merged**